compiler/testSuite/Uwhile00.bal

```diff
@@ -0,0 +1,5 @@
+public function main() {
```

Comment:
A U case like a V case. With a println @output etc.  When I implement it, I should be able to rename U to V and it should work.